data/sql/updates/pending_db_world/2016_08_19_00.sql

```diff
@@ -0,0 +1,31 @@
+
+/*
+2016-08-16 00:44:35 ERROR: Table 'creature_loot_template' entry 1 isn't creature entry and not referenced from loot, 
+and then useless.
+
+Removed, items inside are lootable from no one mobs.
+*/
+DELETE FROM creature_loot_template WHERE entry = 1;
+DELETE FROM conditions WHERE SourceTypeOrReferenceId = 1 AND SourceGroup = 1;
```

Comment:
Please, investigate on it before, because seems that they used that loots for some map scritps ( take a look at conditions that you removed )

However creature 1 is the waypoint creature present in creature_template, don't you have it?